tests/test_countrymask.py

```diff
@@ -132,5 +153,16 @@ def test_country_match(country_codes,not_country_codes,fp_directory):
         country_code_match = [not_c_code for c in ds_both['country_code'].values if not_c_code == c]
         
         assert len(country_code_match) == 0
+        
+@pytest.mark.skipif(not glob.glob(os.path.join(data_path,"World_shape_databases")), reason="No access to files in data_path.")
```

Comment:
Do you need this skipif for this test?

Reply:
Good point. I've removed that skip statement.